config/pipelines/high_throughput_scrna_core_cell_extraction.wip.yml

```diff
@@ -0,0 +1,22 @@
+---
+# Initial transfer from tube to plate, as entry for one or other of below pipelines. Happens without submissions / requests.
+scRNA Core Cell Extraction Entry:
+  pipeline_group: scRNA Core Cell Extraction
+  relationships:
+    LRC Blood Vac: LRC Blood Bank
+# scRNA Core Cell Extraction pipeline
+scRNA Core Cell Extraction Seq:
+  pipeline_group: scRNA Core Cell Extraction
+  filters: &scrna_core_cell_extraction_filters
+    request_type_key: limber_scrna_core_cell_extraction
+  relationships:
+    # LRC Blood Vac: LRC Blood Bank
```

Comment:
Is there some uncertainty here? Could these various commented parts go?

Reply:
Oh well done for finding this - looks like that second commit was the "weird technical reason" I was looking for... we may well find that we need to add this line back in for the same reason, when we get to DPL-811.